src/hooks/userTracking/useUserTracking.ts

```diff
@@ -5,12 +5,27 @@ import {
   TrackingEventParameter,
 } from '@/const/trackingKeys';
 import { useAccounts } from '@/hooks/useAccounts';
-import type { TrackEventProps } from '@/types/userTracking';
+import { useJumperTracking } from '@/hooks/useJumperTracking';
+import { useSession } from '@/hooks/useSession';
+import type {
+  TrackEventProps,
+  TrackTransactionProps,
+} from '@/types/userTracking';
 import { EventTrackingTool } from '@/types/userTracking';
+import type { Theme } from '@mui/material';
+import { useMediaQuery } from '@mui/material';
 import { useCallback, useEffect } from 'react';
+import { useIntegrator } from '../useIntegrator';
 
 export function useUserTracking() {
   const { account } = useAccounts();
+  const isDesktop = useMediaQuery((theme: Theme) => theme.breakpoints.up('md'));
+  const sessionId = useSession();
+  const integratorString = useIntegrator();
```

Comment:
What is the objective of this? We should have the integrator returned into the event already

Reply:
Great point, yes since this is needed in tracking but also can vary, I added this hook. But taking it directly from the widget-events seems to be the best solution!